admin-action/package.json.liquid

```diff
@@ -6,21 +6,21 @@
 {%- if flavor contains "preact" %}
   "dependencies": {
     "preact": "^10.10.x",
-    "@shopify/ui-extensions": "~2025.10.0-rc"
+    "@shopify/ui-extensions": "2024.07.x"
```

Comment:
do you really want to change the preact version? That only works with remote-dom, eh?